pkgs/top-level/all-packages.nix

```
@@ -11106,6 +11106,8 @@ with pkgs;
 
   wf-recorder = callPackage ../applications/video/wf-recorder { };
 
+  whatweb = callPackage ../tools/security/whatweb { };
```

```suggestion
  whatweb = callPackage ../tools/security/whatweb { };
```
I don't see how this would be used for security.

---

It is used for pentesting

---

Also, it is listed at #81418.